onadata/apps/logger/models/xform_submissions_counter.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from django.contrib.auth.models import User
+from django.contrib.gis.db import models
+from django.utils import timezone
+
+from onadata.apps.logger.models.xform import XForm
+
+
+class XFormSubmissionCounter(models.Model):
```

Comment:
in light of https://github.com/kobotoolbox/kpi/issues/3795, we may want to store daily counters for at least the past 31 days, maybe longer. @noliveleger and I are discussing and will revisit this